### PR TITLE
Readme.md changes for itd_service

### PR DESCRIPTION
**Pull request by saichint**
This PR is for documenting the pre-requisites for cisco_itd_service.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2265,6 +2265,7 @@ Manages configuration of ITD (Intelligent Traffic Director) service.
 
 | Property | Caveat Description |
 |:--------|:-------------|
+| `Entire` <br> `provider` <br> | For this provider to work, CLI commands need to be executed <br> via command_config prior to invoking the manifest. They are <br> 'feature pbr' <br> 'feature sla sender' <br> 'feature sla responder' <br> 'ip sla responder' |
```
Comment (mikewiebe) [inline README.md]: How about this instead?

| Property | Caveat Description |
|:------------|:--------------------------|
| | This provider requires the following commands to be applied as prerequisites using the [cisco_command_config](https://github.com/cisco/cisco-network-puppet-module/blob/master/README.md#type-cisco_command_config) provider.<br><br>&nbsp;&nbsp;cisco_command_config { 'prerequisites':<br>&nbsp;&nbsp;&nbsp;&nbsp;command => "<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature pbr<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature sla sender<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature sla responder<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ip sla responder<br>&nbsp;&nbsp;&nbsp;&nbsp;"<br>&nbsp;&nbsp;}|

Comment (mikewiebe) [inline README.md]: Please test that you can literally cut and paste this into a site.pp file and apply to make it easier for users.

File: README.md [hunk 2]
```diff
@@ -2265,6 +2265,7 @@ Manages configuration of ITD (Intelligent Traffic Director) service.
 
 | Property | Caveat Description |
 |:--------|:-------------|
+|  | This provider requires the following commands to be applied as prerequisites using the [cisco_command_config](https://github.com/cisco/cisco-network-puppet-module/blob/master/README.md#type-cisco_command_config) provider.<br><br>&nbsp;&nbsp;cisco_command_config { 'prerequisites':<br>&nbsp;&nbsp;&nbsp;&nbsp;command => "<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature pbr<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature sla sender<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;feature sla responder<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ip sla responder<br>&nbsp;&nbsp;&nbsp;&nbsp;"<br>&nbsp;&nbsp;}|
 | `nat_destination` | Supported only on N7k |
```
Comment (mikewiebe) [inline README.md]: Couple additional questions:
- Do all of these commands need to be applied on both `n7k` and `n9k`?
- Did you test them on both?
- Does this apply to any of the other itd providers?

Comment (saichint) [inline README.md]: * Yes. we need to apply the commnds to both n7k and n9k
* Yes, I tested on both platforms
* There are 3 itd providers. itd_device_group and itd_device_group_node and itd_service. The first 2 can be configured but they have no real meaning until itd_service is configured (in the itd_service manifest, user specifies which itd_device_group will be used) so if we run the prerequisite before itd_service, we should be ok.

Comment (mikewiebe) [inline README.md]: 👍

**Pull request merged**